NeuroRuler/GUI/main.py

```diff
@@ -847,6 +851,49 @@ def export_curr_slice_as_img(self, extension: str) -> None:
         )
         self.image.pixmap().save(path, extension)
 
+    def export_json(self) -> None:
+        """Called when "export" button is clicked
+        
+        :return: `None`"""
+        file_name = (
+            global_vars.CURR_IMAGE_INDEX + 1
+            if settings.EXPORTED_FILE_NAMES_USE_INDEX
+            else get_curr_path().name
+        )
+        path: str = str(
+            constants.JSON_DIR
+            / f"{file_name}.json"
+        )
+
+        output_path: str = str(constants.JSON_DIR)
+        
+        circumference = self.get_circumference()
+
+        data = {
+        "Image_name": file_name,
+        "output_folder": output_path,
+        "x_rotation": global_vars.THETA_X,
+        "y_rotation": global_vars.THETA_Y,
+        "z_rotation": global_vars.THETA_Z,
+        "slice": global_vars.SLICE,
+        "smoothing_conductance": global_vars.CONDUCTANCE_PARAMETER,
+        "smoothing_iterations": global_vars.SMOOTHING_ITERATIONS,
+        "smoothing_time_step": global_vars.TIME_STEP,
+        "circumference": circumference
+        }
+
+        with open(path, 'w') as outfile:
+            json.dump(data, outfile)
+
+    def get_circumference(self) -> str:
+        """Get the value on the "circumference_label"
+
+        :return: `str`"""
+        circumference_label: str = self.circumference_label.text()
+        split_parts = re.split('[:(]', circumference_label)
+        circumference = split_parts[1].strip()
+        return circumference
```

Comment:
Is there a less hacky way to store the circumference data?

Reply:
Good point. Let me improve this in the next commit.